fileupload.go

```diff
@@ -12,18 +12,18 @@ import (
 // file upload.
 // For more details see https://stripe.com/docs/api#create_file_upload.
 type FileUploadParams struct {
-	Params  `form:"*"`
-	Purpose FileUploadPurpose
-
-	// Filename is just the name of the file without path information.
-	Filename string
+	// File is a deprecated form of FileReader and Filename that will do the same thing, but
+	// allows referencing a file directly. Please prefer the use of FileReader and Filename instead.
+	File *os.File
 
 	// FileReader is a reader with the contents of the file that should be uploaded.
 	FileReader io.Reader
 
-	// File is a deprecated form of FileReader and Filename that will do the same thing, but
-	// allows referencing a file directly. Please prefer the use of FileReader and Filename instead.
-	File *os.File
+	// Filename is just the name of the file without path information.
+	Filename string
+
+	Params  `form:"*"`
```

Comment:
In other classes you kept Params at the top. I think it's "worth" keeping always at the top since it's not really a parameter